sentry/src/main/java/io/sentry/environment/SentryEnvironment.java

```diff
@@ -20,7 +19,7 @@
     /**
      * Version of this SDK.
      */
-    public static final String SDK_VERSION = ResourceBundle.getBundle("sentry-build").getString("build.name");
+    public static final String SDK_VERSION = Version.SDK_VERSION;
```

Comment:
If you're doing this globally (i.e., on the JVM as well as on Android), shouldn't you be removing the bundle and the build-time processing of it?  Or are you adding this as an alternate option only, or doing a staged deprecation?

Reply:
Ah good point, I guess it's pretty big reach to think someone is manually using our resource for anything. It seems harmless to leave in (as you said, maybe until next major version) or I can just remove it. I'm indifferent tbh.

Reply:
Yeah, on a second pass since I'm re-using `$buildNumber` anyway it seems harmless to leave the properties file there until we do a real deprecation.